alarms/dojot.auth.AuthenticationError.xml

```diff
@@ -0,0 +1,12 @@
+<alarm-metamodel>
+       <domain>AuthenticationError</domain>
+       <namespace>dojot.auth</namespace>
+       <primary-subject>
+               <primary-field type="alphanumeric">module_name</primary-field>
+               <primary-field type="alphanumeric">instance_id</primary-field>
+       </primary-subject>
+       <additional-data>
+               <additional-field type="numeric">process_id</additional-field>
+               <additional-field type="numeric">signal</additional-field>
```

Comment:
Is this a signal like in `SIGHUP`, `SIGKILL`, and so on?

Reply:
Yes, I will change with the other suggestions